src/usage_metrics/scripts/save_kaggle_metrics.py

```diff
@@ -15,10 +16,17 @@
 def get_kaggle_dataset_metadata() -> str:
     """Get PUDL project usage metadata from Kaggle site."""
     api = KaggleApi()
-    kaggle_owner = "catalystcooperative"
-    kaggle_dataset = "pudl-project"
-
-    metadata = api.metadata_get(kaggle_owner, kaggle_dataset)
+    api.authenticate()
+    api.dataset_metadata(
+        "catalystcooperative/pudl-project", Path.cwd()
+    )  # Download metrics to CWD.
+
+    json_path = Path(Path.cwd(), "dataset-metadata.json")
+    with Path.open(json_path) as file:
+        metadata_str = json.load(
+            file
+        )  # Kaggle downloads to the current working directory.
+        metadata = json.loads(metadata_str)  # Fix bad formatting in original JSON file
```

Comment:
ugh that's kinda annoying to have to download the file and then load it.. but alas. should we delete the downloaded file afterwards or add it to a gitignore?